app/app.go

```diff
@@ -846,5 +906,48 @@ func (app *App) registerUpgradeHandlers() error {
 		app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
 	}
 
+	app.UpgradeKeeper.SetUpgradeHandler("v2.2.0-alpha1", func(ctx sdk.Context, plan upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
+		// 1st-time running in-store migrations, using 1 as fromVersion to
+		// avoid running InitGenesis.
+		fromVM := map[string]uint64{
+			"auth":         1,
+			"bank":         1,
+			"capability":   1,
+			"crisis":       1,
+			"distribution": 1,
+			"evidence":     1,
+			"gov":          1,
+			"mint":         1,
+			"params":       1,
+			"slashing":     1,
+			"staking":      1,
+			"upgrade":      1,
+			"vesting":      1,
+			"ibc":          1,
+			"genutil":      1,
+			"transfer":     1,
+			"aol":          1,
+			"did":          1,
+			"burn":         1,
+			"wasm":         1,
+			"datadeal":     1,
+			"datapool":     1,
+			"oracle":       1,
+		}
+
+		app.IBCKeeper.ConnectionKeeper.SetParams(ctx, ibcconnectiontypes.DefaultParams())
+
+		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
+	})
+
+	if upgradeInfo.Name == "v2.2.0-alpha1" && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
+		storeUpgrades := storetypes.StoreUpgrades{
+			Added: []string{"authz", "feegrant"},
```

Comment:
Yes  you're right. Because it's the first time to add the modules you mentioned.
And, I forgot to delete `token` module in this `StoreUpgrades`. I'll add it.
Thx!

Reply:
fixed in [516fc95](https://github.com/medibloc/panacea-core/pull/356/commits/516fc954214675cf8659cc8802202e09059d1b8a)